### PR TITLE
Documentation update of ctcc/nrf52840 and adding it to release notes

### PR DESCRIPTION
**Pull request by kowalewskijan**
Provides a picture of M.2 version of a card and fixes quality of mPCIe version.
Adds ctcc/nrf52840 to release notes.

### CODE REVIEW

**Review by aescolar — COMMENTED** [review]
That's an extra 170KB of binary data in the tree. The original picture seems quite enough for Zephyr's documentation.
@kartben do we have some policy about this?

**Review by aescolar — DISMISSED** [review]
If you could remove the pictures change and leave only release notes change we could merge the release notes change right away and defer the discussion about the new pictures. Thank you.

File: boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png
Comment (nordicjm) [inline boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png:1]: Remove excess spacing on images, convert to webp and put through https://tinypng.com/

**Review by nordicjm — CHANGES_REQUESTED** [review]

File: boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png
Comment (kowalewskijan) [reply boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png:1]: is it really necessary to have them in webp? I think it is allowed to have `.png`, `.jpeg`, `.jpg` or `.svg`

**Comment by kowalewskijan** [comment]
> If you could remove the pictures change and leave only release notes change we could merge the release notes change right away and defer the discussion about the new pictures. Thank you.

According to compliance check, each picture can have up to 100kB. Right now the current picture seems really blurred and since the documentation mentions also M.2 version I wanted to add a picture of such version also.

File: boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png
Comment (kartben) [reply boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png:1]: the point is to try to use as modern of a file format as possible, and webp is typically the best for boards images where we kind of want the best of both PNG (with ability to have a transparent background) and JPG (overall compression)

**Comment by kartben** [comment]
> > If you could remove the pictures change and leave only release notes change we could merge the release notes change right away and defer the discussion about the new pictures. Thank you.
> 
> @aescolar According to compliance check, each picture can have up to 100kB. Right now the current picture seems really blurred and since the documentation mentions also M.2 version I wanted to add a picture of such version also.

The images you contributed are also quite blurry, fwiw, and probably mainly because you had to make them pretty small, pixel-wise, to fit under 100K.

This one https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/34de9cc150a9e88851b486b92fd00429b7e9add1/boards/ct/ctcc/doc/img/ctcc_nrf52840_mpcie.png , besides not having a transparent background, is basically only 13K if converted to Webp

I would recommend you try to use your original hi-res 3D renders (if you have access to them) and convert them to 600px wide webp image. This way you'll have a transparent background, and best visual quality.

TLDR: yes compliance check says 100K max but above all we want to be mindful of the fact these are resources that end up making the Git repo increasingly large, so each new image being introduced should be "worth it".

Thanks!

File: boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png
Comment (kowalewskijan) [reply boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.png:1]: Ok, thanks. I'll use original renders and convert them to webp

**Comment by kowalewskijan** [comment]
@aescolar @kartben @nordicjm - I converted images to webp and to equal size of 600x800 which gave 32kB each. Let me know if it's fine.

**Comment by aescolar** [comment]
@kowalewskijan would it make sense to crop all that white away?

**Comment by kowalewskijan** [comment]
Pictures have transparent background, I checked that by viewing `index.rst`

**Comment by kowalewskijan** [comment]
@kartben @nordicjm PTAL

**Review by kartben — DISMISSED** [review]

File: boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.webp
Comment (nordicjm) [inline boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.webp:1]: Why is there such excess spacing on these images? It looks very silly on https://builds.zephyrproject.io/zephyr/pr/76107/docs/boards/ct/ctcc/doc/index.html

I imagine it makes the pdf look worse

Comment (kowalewskijan) [reply boards/ct/ctcc/doc/img/ctcc_nrf52840_m2.webp:1]: I trimmed the excessive spacing around images. PTAL

**Review by nordicjm — APPROVED** [review]

**Comment by kowalewskijan** [comment]
@kartben, could you take a look one more time?

**Review by kartben — APPROVED** [review]
